### PR TITLE
Bookend interaction changes

### PR DESCRIPTION
**Pull request by alanorozco**
## Exit/entrance transition
1 transition

## Card goes into full bleed after a certain scroll threshold
2 full bleed

## Share widget clipped horizontally on card/full bleed
3 fullbleed clipping

## Bookends that won't fill the screen will appear on the bottom
4 small

### CODE REVIEW

File: extensions/amp-story/0.1/amp-story.css [hunk 1]
```diff
@@ -115,6 +114,10 @@ amp-story[standalone]:fullscreen {
   background-color: rgba(0, 0, 0, 0.2) !important;
 }
 
+.i-amphtml-story-system-layer .i-amphtml-story-button {
+  float: left;
```
Comment (newmuis) [inline extensions/amp-story/0.1/amp-story.css]: `!important` any styles that shouldn't be overridden; if they are allowed to be overridden, move them to the separate CSS file.

File: extensions/amp-story/0.1/amp-story.css [hunk 2]
```diff
@@ -492,7 +550,7 @@ amp-story-grid-layer * {
 }
 
 .i-amphtml-story-share-icon:active::before {
-  background-color: rgba(255, 255, 255, 0.6) !important;
+  background-color: rgba(120, 120, 120, 1) !important;
```
Comment (newmuis) [inline extensions/amp-story/0.1/amp-story.css]: Why not `#787878`?

File: extensions/amp-story/0.1/amp-story.css [hunk 3]
```diff
@@ -476,13 +533,14 @@ amp-story-grid-layer * {
 .i-amphtml-story-share-icon::before {
   content: " ";
   border-radius: 4px !important;
-  background-color: rgba(255, 255, 255, 0.3) !important;
+  background-color: rgba(33, 33, 33, 1) !important;
```
Comment (newmuis) [inline extensions/amp-story/0.1/amp-story.css]: Why not `#212121`?

File: extensions/amp-story/0.1/amp-story.css [hunk 4]
```diff
@@ -704,9 +762,32 @@ amp-story[standalone] .i-amphtml-story-developer-log {
...
+  top: 8px !important;
+  left: 8px !important;
+  position: absolute !important;
   background-image: url('data:image/svg+xml;charset=utf-8,<svg xmlns="http://www.w3.org/2000/svg" fill="#FFFFFF" height="24" viewBox="0 0 24 24" width="24"><path d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"/><path d="M0 0h24v24H0z" fill="none"/></svg>') !important;
 }
 
+.i-amphtml-story-bookend-fullbleed .i-amphtml-story-bookend-close {
+  opacity: 1 !important;
+  position: fixed !important;
+  z-index: 2;
```
Comment (newmuis) [inline extensions/amp-story/0.1/amp-story.css]: Are publishers meant to override this?

File: extensions/amp-story/0.1/amp-story.css (hunk 1 above)
Comment (alanorozco) [reply extensions/amp-story/0.1/amp-story.css]: Done.

File: extensions/amp-story/0.1/amp-story.css (hunk 4 above)
Comment (alanorozco) [reply extensions/amp-story/0.1/amp-story.css]: Nope, fixed.

File: extensions/amp-story/0.1/amp-story.css (hunk 3 above)
Comment (alanorozco) [reply extensions/amp-story/0.1/amp-story.css]: Consistency, but it's nbd. Changed.

File: extensions/amp-story/0.1/amp-story.css (hunk 2 above)
Comment (alanorozco) [reply extensions/amp-story/0.1/amp-story.css]: Ditto

**Pull request merged**